server/templates/onboarding_page.html

```diff
@@ -27,11 +27,19 @@ <h3>Step 1 — Download your transcript</h3>
               Quest</a> and follow the instructions <a target="_blank" href="https://uwaterloo.ca/quest/help/students/how-do-i/unofficial-transcript">here</a> to download your transcript as a PDF.
           </p>
           <br/>
-          <h3>Step 2 — Open and copy your transcript</h3>
+          <h3>Step 2 — Open and copy your transcript in Google Chrome</h3>
```

Comment:
This should probably be parametrized by Chrome/not Chrome too if you want to change it.

Reply:
@CatastrophiClam